apps/condo/domains/resident/schema/RegisterResidentService.js

```diff
@@ -32,12 +39,19 @@ const RegisterResidentService = new GQLCustomSchema('RegisterResidentService', {
                     user: { connect: { id: context.authedItem.id } },
                 }
                 const [existingResident] = await ResidentAPI.getAll(context, {
-                    address,
+                    address_i: address,
```

Comment:
It seems, like this criteria will bring false positive results, for example in case, when we are going go register a new resident with address "Some street 1" and another resident lives on "Some street 12", so a new resident will be handled as existing one.